mmv1/third_party/terraform/acctest/bootstrap_test_utils.go

```diff
@@ -1297,3 +1299,37 @@ func SetupProjectsAndGetAccessToken(org, billing, pid, service string, config *t
 
 	return accessToken, nil
 }
+
+type BootstrappedIntegrationsClient struct {
+	Name string
+}
+
+func BootstrapIntegrationsClient(t *testing.T, region string) BootstrappedIntegrationsClient {
+	config := BootstrapConfig(t)
+	if config == nil {
+		return BootstrappedIntegrationsClient{}
+	}
+
+	integrationsClient := config.NewIntegrationsClient(config.UserAgent)
+
+	project := envvar.GetTestProjectFromEnv()
+	parent := fmt.Sprintf("projects/%s/locations/%s", project, region)
```

Comment:
We're failing on `GET /v1/projects/ci-test-project-188019/locations/us-west1/clientmetadata`. I don't see that method under https://cloud.google.com/application-integration/docs/reference/rest, but looking at the client library [source code](https://github.com/googleapis/google-api-go-client/blob/bf1c054cabf336790585202417c35f8801a5c1a9/integrations/v1/integrations-gen.go#L11995) it looks like we want to be calling `GET /v1/projects/{projectsId}/clientmetadata`

```suggestion
	parent := fmt.Sprintf("projects/%s", project)
```

Reply:
It seems like the client library is outdated which is causing too much confusions. I am working with the team to update that. For now, In order to unblock, I have removed the bootstrapping code and added clients as resource with different regions. I would add the bootstrapping back in future once the client library is updated.